app/src/main/res/values-ca/strings.xml

```diff
@@ -3,80 +3,31 @@
   <string name="action_cancel">Cancel·la</string>
```

Comment:
I don't know if this deletion is necessary - this should be done with a string export rather than a manual? deletion, because we should export so it doesn't get too out of date with the l10n repo (though actually, maybe it's fine because this is mainly a string deletion?). And we should do a string export/import without any new strings so Delphine doesn't need to look at a horrible diff with new strings and deleted strings.

Reply:
@liuche I don't understand the l10n system too well so I don't know. What do you propose?

I'd think it's better to give us a clean repository slate for Firefox TV, which is what I'm trying to do here. I figured the import/export tools would just delete all these already-translated strings. I agree with the idea that we should just export this so it's easier for the l10n team.